src/main/java/com/powsybl/openloadflow/network/impl/extensions/OverloadManagementSystem.java

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.openloadflow.network.impl.extensions;
+
+/**
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
+ */
+public interface OverloadManagementSystem {
+
+    boolean isEnabled();
+
+    void setEnabled(boolean enabled);
+
+    String getLineIdToMonitor();
```

Comment:
Line or Branch ?

Reply:
`monitoredBranch`?

Reply:
From my knowledge, this kind of system are only used on lines

Reply:
MonitoredLine?

Reply:
ok

Reply:
Can you add that you look at side 1?

Reply:
We should be able to configure it. This is a first and simple version.